package.json

```diff
@@ -74,9 +74,9 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.23.2",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-unicorn": "^31.0.0",
-    "jest": "^26.6.3",
-    "prettier": "^2.2.1",
+    "eslint-plugin-unicorn": "^34.0.1",
```

Comment:
these had breaking changes, and had to use `npm install -D {package}@latest`. I checked, nothing breaks, and a skimmed through the changelog (things look fine).